ansible/vars/aeternity/dev2.yml

```diff
@@ -1,3 +1,5 @@
+public_ipv4: "{{ ansible_ec2_public_ipv4|default(ansible_ssh_host)|default(ansible_host)|default(inventory_hostname) }}"
+datadog_enabled: yes
```

Comment:
It should be enabled on dev2

Reply:
There is no group_vars for dev2, so I disabled it

Reply:
ehm, yes I meant it should be disabled :)